src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
use classes 

u can use tags to reset margins, but all other styles shouldn't be here

Reply:
done